atom/browser/api/atom_api_web_contents.cc

```diff
@@ -822,14 +828,17 @@ void WebContents::AddNewContents(content::WebContents* source,
   }
 
   if (blocked) {
-    auto guest = brave::TabViewGuest::FromWebContents(new_contents);
+    auto guest = brave::TabViewGuest::FromWebContents(new_contents.get());
     if (guest) {
```

Comment:
@hferreiro @ltilve this causes a double delete because the unique_ptr tries to delete the webcontents again when it goes out of scope

Reply:
the logic of this whole thing is a bit convoluted now, but @darkdh will do a quick fix to release before this delete is called